warehouse/models/mart/transit_database/dim_mobility_mart_providers.sql

```diff
@@ -0,0 +1,160 @@
+{{ config(materialized='table') }}
+
+-- An Organization (i.e. dim_organizations) can provide multiple different
+-- Services (i.e. dim_services). Each Service will have its own funding sources,
+-- GTFS feeds, and target counties.
+
+WITH organizations AS (
+   SELECT * FROM {{ ref('dim_organizations') }}
+   WHERE _is_current
+),
+
+services AS (
+    SELECT * FROM {{ ref('dim_services') }}
+    WHERE _is_current
+),
+
+county_geogs AS (
+    SELECT * FROM {{ ref('dim_county_geography') }}
+),
+
+orgs_hq_bridge AS (
+    SELECT * FROM {{ ref('bridge_organizations_x_headquarters_county_geography') }}
+),
+
+orgs_services_bridge AS (
+    SELECT * FROM {{ ref('bridge_organizations_x_services_managed') }}
+    WHERE _is_current
+),
+
+services_funding_bridge AS (
+    SELECT * FROM {{ ref('bridge_services_x_funding_programs') }}
+    WHERE _is_current
+),
+
+gtfs_services AS (
+    SELECT * FROM {{ ref('dim_gtfs_service_data') }}
+    WHERE _is_current
+),
+
+gtfs_datasets AS (
+    SELECT * FROM {{ ref('dim_gtfs_datasets') }}
+    WHERE _is_current
+),
+
+-- This query will collect every funding source an Organization has via the
+-- services it provides.
+funding_by_organization AS (
+    SELECT
+        orgs.key AS org_key,
+        ARRAY_AGG(services_funding_bridge.funding_program_name) AS funding_sources
+    FROM
+        organizations orgs
+    INNER JOIN
+        orgs_services_bridge ON orgs_services_bridge.organization_key = orgs.key
+    INNER JOIN
+        services_funding_bridge ON orgs_services_bridge.service_key = services_funding_bridge.service_key
+    GROUP BY
+        orgs.key
+),
+
+-- This table has historical data for every year (looks like dating back to
+-- 2018), so let's narrow down that table to *just* the most recent year so
+-- we don't get duplicate rows for each year.
+--
+-- We cannot use `_is_current` here because every year is marked as "current"
+-- since it's the "current" record for the respective year.
+annual_ntd AS (
+    SELECT * FROM {{ ref('dim_annual_ntd_agency_information') }}
+    WHERE state = "CA"
+
+    -- We only want data from the latest data from NTD. In the rare edge case
+    -- that an NTD ID is removed and no longer exists in the current year's
+    -- dataset, then replace `DENSE_RANK()` with `ROW_NUMBER()` and add a
+    -- `PARTITION BY ntd_id` inside of the `OVER()`.
+    QUALIFY DENSE_RANK() OVER(ORDER BY year DESC) = 1
+),
+
+gtfs_data_by_organization AS (
```

Comment:
I think you might be able to replace this (and maybe the orgs to services join above as well?) with a reference to [`dim_provider_gtfs_data`](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.dim_provider_gtfs_data) which might just simplify things a little (.... sorry for not suggesting this earlier 😅)